README.md

```diff
@@ -174,3 +174,34 @@ The integration will close the active session.
 ```js
 player.load({…});
 ```
+
+#### External VST tracking
+
+If your app needs additional setup steps which should be included in VST tracking, such as DRM token generation, before the `Player` instance can be initialized, 
+the `ConvivaAnalytics` can be initialized without a `Player` instance. Once the `Player` instance is created it can be attached.
+
+1. Create the `ConvivaAnalytics` instance with your `customerKey` and configuration.
+
+```typescript
+const convivaAnalytics = new ConvivaAnalytics(undefined, customerKey);
```

Comment:
Is there a possibility in typescript to omit the `undefined` parameter here? that a customer only need to do `new ConvivaAnalytics(customerKey);`? Or is that not possible as the `player` is the first parameter?

Reply:
There is constructor overloading in TS, but I don't think it's possible in this case:

<img width="1059" alt="image" src="https://github.com/user-attachments/assets/10620b59-2cc4-4986-afc0-e5fc4622ad35">